docs/api-guide.md

```diff
@@ -773,117 +829,111 @@ Scan scanUsingSpecifiedNamespace =
         .build();
 ```
 
-#### Notes
-
-Although all the builders of the CRUD operations can specify consistency by using the `consistency()` methods, those methods are ignored. Instead, the `LINEARIZABLE` consistency level is always used in transactions.
-
 ### Commit a transaction
 
 After executing CRUD operations, you need to commit a transaction to finish it.
 
-You can commit a transaction as follows;
+You can commit a transaction as follows:
 
 ```java
-// Commit a transaction
+// Commit a transaction.
 transaction.commit();
 ```
 
-### Rollback/Abort a transaction
+### Roll back or abort a transaction
 
-If you want to rollback/abort a transaction or an error happens during the execution, you can rollback/abort a transaction.
+If an error occurs when executing a transaction, you can roll back or abort the transaction.
 
-You can rollback/abort a transaction as follows;
+You can roll back a transaction as follows:
 
 ```java
-// Rollback a transaction
+// Roll back a transaction.
 transaction.rollback();
+```
 
-Or
+Or, you can abort a transaction as follows:
 
-// Abort a transaction
+```java
+// Abort a transaction.
 transaction.abort();
 ```
 
-Please see [Handle exceptions](#handle-exceptions) for the details of how to handle exceptions in ScalarDB.
+For details about how to handle exceptions in ScalarDB, see [How to handle exceptions](#how-to-handle-exceptions).
 
-## Handle exceptions
+## How to handle exceptions
 
-Handling exceptions correctly in ScalarDB is very important.
-If you mishandle exceptions, your data could become inconsistent.
-This document explains how to handle exceptions properly in ScalarDB.
+When executing a transaction, you will also need to handle exceptions properly. If you don't handle exceptions properly, you may face anomalies or data inconsistency.
```

Comment:
IMO, since anomalies or data inconsistency, are critical issues for ScalarDB users, it would be worth emphasizing those risks. (with `notice` section?)

Reply:
Great idea! I've added an "Attention" block since this is important for users to know. The following is how the "Attention" block will look on the docs site.

![how-to-handle-exceptions-attention-block](https://github.com/scalar-labs/scalardb/assets/23216828/dc1427c8-04f4-42d2-9a99-887da88401f2)

Some other docs have the same sentence, so I'll make the same change in those docs.